### PR TITLE
Allow turbolift specific-floor delays

### PR DESCRIPTION
**Pull request by Arokha**
If one floor is 'taller' than the others, or otherwise takes longer to traverse, you can set delay_time on the area to have it wait a different time there. Or, wait even less time, though less than 1 second isn't possible since the controller only fires every second.

Also makes the default elevator panel nanoui window sliiightly bigger. It often had scrollbars for me after a few floors.

### CODE REVIEW

**Comment by Arokha** [comment]
This is failing Travis because Travis is using a version of Byond to compile it that is too old to understand that expression. I'll make a PR to update Travis.

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ language: c
 sudo: false
 
 env:
-  BYOND_MAJOR="510"
-  BYOND_MINOR="1346"
-  MACRO_COUNT=986
+  BYOND_MAJOR="511"
+  BYOND_MINOR="1381"
+  MACRO_COUNT=875
```
Comment (PsiOmegaDelta) [inline .travis.yml]: Sneaky version update.

Comment (Arokha) [inline .travis.yml]: What do you mean? I left a comment about it, and it has it's own commit. Wasn't trying to hide it or anything.

Comment (Anewbe) [inline .travis.yml]: I knew it was happening, and I thought I saw those mentioned things somewhere.

**Pull request merged**